### PR TITLE
Changes to the java-maven Dockerfile

### PR DESCRIPTION
**Pull request by JWGmeligMeyling**
Changes required to get the `java-maven` dockerfile to work with Docker 1.3.3

### CODE REVIEW

File: build-server/src/main/resources/dockerfiles/java-maven/Dockerfile [hunk 1]
```diff
@@ -27,7 +26,7 @@ RUN ln -s /usr/share/maven3/bin/mvn /usr/bin/mvn
 RUN apt-get -y install psmisc xfonts-100dpi xfonts-75dpi xfonts-scalable xfonts-cyrillic xvfb libxrender-dev libxtst6 libxi6
 
 RUN echo '#!/bin/bash' >> /usr/bin/with-xvfb
-RUN echo '/usr/bin/Xvfb :99 -ac -screen 0 1024x768x8&' >> /usr/bin/with-xvfb
+RUN echo '/usr/bin/Xvfb :99 -ac -screen 0 1024x768x8 2>&1 2>/dev/null &' >> /usr/bin/with-xvfb
```
Comment (JWGmeligMeyling) [inline build-server/src/main/resources/dockerfiles/java-maven/Dockerfile]: Send `Xvfb` logs to `/dev/null`

File: build-server/src/main/resources/dockerfiles/java-maven/Dockerfile [hunk 2]
```diff
@@ -36,11 +35,17 @@ RUN echo 'exit $exit_value' >> /usr/bin/with-xvfb
 RUN chmod +x /usr/bin/with-xvfb
 
 RUN echo '#!/bin/bash' >> /usr/bin/fetch-deps
-RUN echo 'git clone https://github.com/devhub-tud/ti2206-template.git' >> /usr/bin/fetch-deps
-RUN echo 'cd ti2206-template' >> /usr/bin/fetch-deps
-RUN echo 'with-xvfb mvn -B clean package' >> /usr/bin/fetch-deps
+RUN echo 'git clone https://github.com/SERG-Delft/jpacman-framework.git' >> /usr/bin/fetch-deps
+RUN echo 'cd jpacman-framework' >> /usr/bin/fetch-deps
+RUN echo 'git checkout 5f132d16d9258026ab1a19a973c23f275c1ad0e9' >> /usr/bin/fetch-deps
+RUN echo 'with-xvfb mvn -B clean install' >> /usr/bin/fetch-deps
```
Comment (JWGmeligMeyling) [inline build-server/src/main/resources/dockerfiles/java-maven/Dockerfile]: Install `jpacman-framework` (as the repository is down). Version 6.0.0 is checked out.

File: build-server/src/main/resources/dockerfiles/java-maven/Dockerfile [hunk 3]
```diff
@@ -36,11 +35,17 @@ RUN echo 'exit $exit_value' >> /usr/bin/with-xvfb
...
 RUN echo 'cd ../' >> /usr/bin/fetch-deps
-RUN echo 'rm -rf ti2206-template' >> /usr/bin/fetch-deps
+RUN echo 'rm -rf jpacman-framework' >> /usr/bin/fetch-deps
+RUN echo 'git clone https://github.com/SERG-Delft/jpacman-template.git' >> /usr/bin/fetch-deps
```
Comment (JWGmeligMeyling) [inline build-server/src/main/resources/dockerfiles/java-maven/Dockerfile]: Clone the `jpacman-template` from `SERG-Delft` as the `devhub-tud` fork is removed.

File: build-server/src/main/resources/dockerfiles/java-maven/Dockerfile [hunk 4]
```diff
@@ -12,7 +11,7 @@ RUN add-apt-repository ppa:natecarlson/maven3
 RUN apt-get update && apt-get -y upgrade
 
 # Automatically accept oracle license
-RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
+RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
```
Comment (JWGmeligMeyling) [inline build-server/src/main/resources/dockerfiles/java-maven/Dockerfile]: Needed to change this to get the Dockerfile to build on my Docker version (1.4.0)

File: build-server/src/main/resources/dockerfiles/java-maven/Dockerfile [hunk 5]
```diff
@@ -2,7 +2,6 @@ FROM ubuntu:precise
 
 MAINTAINER Michael de Jong "https://github.com/michaeldejong"
 
-RUN echo "deb http://archive.ubuntu.com/ubuntu precise main universe" > /etc/apt/sources.list
```
Comment (JWGmeligMeyling) [inline build-server/src/main/resources/dockerfiles/java-maven/Dockerfile]: Needed to change this to get the Dockerfile to build on my Docker version (1.4.0)

**Comment by michaeldejong** [comment]
:+1:

**Pull request merged**